lib/cosmos/system/system.rb

```diff
@@ -570,8 +570,8 @@ def reset_variables(filename = nil)
       @ports['REPLAY_API'] = 7877
       @ports['REPLAY_PREIDENTIFIED'] = 7879
       @ports['REPLAY_CMD_ROUTER'] = 7880
-      @ports['DART_DECOM'] = 8777
-      @ports['DART_STREAM'] = 8779
+      @ports['DART_STREAM'] = 8777
+      @ports['DART_DECOM'] = 8779
```

Comment:
This seemed more straight forward than updating the demo and install config and that's probably what people have been using.

Reply:
@ryanatball I'll update the website as well when we merge this. Should I also add a note about the stream server being a low-level interface for COSMOS tools only to avoid confusion? See #1080.

Reply:
This is fine.  The port numbers aren't that important.  They did initially match the JSON-RPC both ending with 777, but I didn't realize the config files were different.  This way existing config files will match the default. 👍